README.md

```diff
@@ -275,3 +275,31 @@ The Azure Diagnostic Log Stream ships events from any files in the `D:\home\LogF
                 flushToDiskInterval: TimeSpan.FromSeconds(1))
             .CreateLogger();
 ```
+
+### Pushing properties to the ILogger
+
+If you want to add extra properties to all logevents in a specific part of your code, you can add them to the **ILogger** in **Microsoft.Extensions.Logging** with the following code. For this code to work, make sure you have added the `.Enrich.FromLogContext()` to the `.UseSerilog(...)` statement, as specified in the samples above.
+
+```csharp
+// Microsoft.Extensions.Logging ILogger
+// Yes, it's required to use a dictionary. See https://nblumhardt.com/2016/11/ilogger-beginscope/
+using (logger.BeginScope(new Dictionary<string, object>
+{
+    ["UserId"] = "xxx",
+    ["ExtraProperty"] = "yyy",
+}))
+{
+   // UserId and ExtraProperty are set for all logging events in these brackets
```

Comment:
The README currently avoids `foo`/`bar`/`xxx` etc., could we try something like:

```csharp
    ["UserId"] = "user-123",
    ["OperationType"] = "update",
```

or something similar?

Reply:
I changed the properties to something more reasonable.